docs/UserGuide.md

```diff
@@ -243,25 +243,34 @@ Examples:<br>`settle 1 amount/500.00`
 
 Deletes the specified student from the address book.
 
-Format: `delete INDEX`
+**Format:** `delete INDEX`
 
-* Deletes the student at the specified `INDEX`.
-* The index refers to the index number shown in the displayed student list.
+**Examples:**
+* `list` followed by `delete 2` deletes the 2nd student in the address book.
+* `find n/Bernice` followed by `delete 1` deletes the 1st student in the results of the `find` command.
+
+**Output:**
+![deleteResult.png](images/deleteResult.png)
+
+<box type="important" header="##### Constraints">
+
+* The index refers to the index number shown in the **displayed** student list.
 * The index **must be a positive integer** 1, 2, 3, …​
```

Comment:
Can capitalize and bold the **INDEX** to be standardized with other command

Reply:
Thanks for catching this! Ive made the changes